Define the role template, different roles and typical tasks (#322)
EVERSE-ResearchSoftware/RSQKit: pages/your_role/principal_investigator.md

start of the project.  

## Software development & quality responsibilities, challenges & tasks

As a Principal Investigator (PI), your responsibilities in relation to software development and quality may include:

- Developing a Software Management Strategy - create a clear plan for how software will be developed, maintained, and 
shared throughout the project. This includes defining sustainability goals, allocating resources, and budgeting appropriately—often formalised in a Software Management Plan required by many funders.
- Incorporating the FAIR research software principles in the software development life-cycle — ensuring that your software 
is Findable, Accessible, Interoperable, and Reusable to support transparency, collaboration, and long-term impact.
European Commission (Horizon Europe) and many national funding bodies strongly encourage adherence to the FAIR principles for both data and software. 
- Establishing roles and responsibilities - identify and assign software-related roles within your team 
(e.g., developer, maintainer, reviewer, documenter, tester, early adopter) to foster accountability, collaboration, and efficient workflows across the project lifecycle.

> - Addressing ethical, environmental, and legal considerations early:

Reviewer: Just a comment - can ignore - ESG is big for companies at the moment - this covers Environmental, Social, and Governance - not sure we want to bring this acronym into the research software space - but I thought worth mentioning.

  - Ethical: promote responsible use of software and data, ensure transparency, and consider contributing improvements or reusable components back to the community.

>   - Environmental: be aware of the energy demands and carbon footprint of your software—opt for efficient algorithms, cloud providers using renewable energy, or local computing where feasible.

Reviewer: Perhaps don't say too much e.g. local computing might not always be great energy efficiency wise - might be worth linking to the task page also - https://everse.software/RSQKit/improving_environmental_sustainability#how-to-monitor-and-reduce-the-environmental-impact-of-my-software

>   - Environmental: be aware of the energy demands and carbon footprint of your software—opt for efficient algorithms, cloud providers using renewable energy, or local computing where feasible.

Author: ```suggestion
  - Environmental: be aware of the energy demands and carbon footprint of your software - e.g. opt for energy-efficient algorithms, cloud providers using renewable energy.
```
